### PR TITLE
Layout improvements

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

- Stretch LayoutSections to be the full width of the place, but have a max width for the content. 
- Replace usages of BasePadding with LayoutSection
- Replace usages of PageWithSidebar with LayoutBase and Drawer. Adds placeholder call-to-action for setting max reserve, creating loan and financing.
- Move create loan/pool submit buttons to below so they don't overlap the wallet button

[#1700](https://github.com/centrifuge/apps/issues/1700)

### Approvals

- [x] Dev

### Screenshots

image

image

image

image

image

### CODE REVIEW

**Comment by sophialittlejohn** [comment]
Seems a bug may have snuck in, the prime detail page doesn't seem to work anymore https://app-pr1773.k-f.dev/prime/aave

**Comment by onnovisser** [comment]
@sophialittlejohn It's in dev too, when you're not connected. This PR #1779 fixes it.

**Review by sophialittlejohn — APPROVED** [review]
Nice! 💯

File: centrifuge-app/src/pages/Loan/index.tsx [hunk 1]
```diff
@@ -136,6 +147,15 @@ const Loan: React.FC<{ setShowOraclePricing?: () => void }> = ({ setShowOraclePr
...
   return (
     <Stack>
+      {loan && !isTinlakeLoan(loan) && (
+        <>
+          <Drawer isOpen={oraclePriceShown} onClose={() => setOraclePriceShown(false)}>
+            <LoadBoundary>
+              <OraclePriceForm loan={loan} />
+            </LoadBoundary>
+          </Drawer>
+        </>
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Loan/index.tsx:157]: unnecessary fragment 🤷

**Review by jpangelle — APPROVED** [review]

**Pull request merged**